2025.02-dev/fpm-alpine/entrypoint.sh

```diff
@@ -165,6 +167,18 @@ if expr "$1" : "apache" 1>/dev/null || [ "$1" = "php-fpm" ]; then
               rsync $rsync_options --ignore-existing /usr/src/friendica/config/ /var/www/html/config/
             fi
 
+            # Add the administrator account
```

Comment:
Same as above. Creating an additional user with 'user add' vs --admin parameter in install_options.

Reply:
Same as above